tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -79,7 +79,7 @@ inline bool ExecuteSingleThreadedGemm(int64_t m, int64_t n, int64_t k,
   constexpr float kHeuristicMultiplier = 1.01;
   const float mul_size = bytes * (m * n + k * (m + n));
   const float l2_heur = l2_size * kHeuristicMultiplier;
-  return mul_size < l2_heur;
+  return (!(mul_size < 0) && (mul_size < l2_heur));
```

Comment:
Nit: I think this is easier to read.
```suggestion
  return (mul_size >= 0 && mul_size < l2_heur);
```

Reply:
@penpornk I will make the change. Thanks for the inputs.